src/biocommons/seqrepo/seqrepo.py

```diff
@@ -269,6 +270,7 @@ def translate_identifier(self, identifier, target_namespaces=None, translate_ncb
     ############################################################################
     # Internal Methods 
 
+    @lru_cache(maxsize=os.environ.get("SEQREPO_LRU_CACHE_MAXSIZE"))
```

Comment:
Prob need to add some documentation about this environment variable to the main readme file.

Reply:
Do you have a suggestion of where? I don't really see anywhere in the README or in the docs that discusses the python usage where this would be useful (maybe it is useful elsewhere but I'm not sure).

Reply:
I think you can a section like the following just above Developing.

```
Environment Variables
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

...
```